README_vi-VN.md

```diff
@@ -0,0 +1,444 @@
+# Hướng dẫn commit message
+
+[![Say Thanks!](https://img.shields.io/badge/Say%20Thanks-!-1EAEDB.svg)](https://saythanks.io/to/RomuloOliveira)
+
+Hướng dẫn này giúp ta hiểu được tầm quan trọng của commit message và cách viết chúng.
+
+Nó giúp bạn biết được commit là gì, tại sao viết một message thì quan trọng, các ví dụ thực tiễn và vài mẹo để lập kế hoạch và viết lại một commit history tốt.
+
+# Ngôn ngữ có sẵn
```

Comment:
Since #31 there is no need to include this section anymore. Could you add in the [README.md](https://github.com/RomuloOliveira/commit-messages-guide/blob/master/README.md) intead?

Reply:
Yes! I fixed it

Reply:
Yes! I have fixed README_vi-VN.md